samples/bluetooth/central_gatt_write/src/central_gatt_write.c

```diff
@@ -35,7 +35,7 @@ static void device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 	}
 
 	/* connect only to devices in close proximity */
-	if (rssi < -70) {
+	if (rssi < -50) {
```

Comment:
There are some bsim tests failing because of changes similar to this. I'm looking into finding where this can be changed